rts/Lua/LuaReadCommon.h

```diff
@@ -0,0 +1,13 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
```

Comment:
Best to name as Recoil, we keep Spring name for historic reasons only, but moving on all new code should use Recoil.